README.md

```diff
@@ -135,6 +135,26 @@ If you are using public cloud, then make sure you see the notes from the Rancher
 
 k3s docs: [k3s configuration / open ports](https://rancher.com/docs/k3s/latest/en/configuration/#open-ports-network-security)
 
+## Working with encrypted ssh keys
```

Comment:
Some people may not understand this terminology. I am sure it is correct, but in layman's terms isn't this an ssh-key with a password? Or `## If your ssh-key is password-protected` ?

Not sure .. would like your input.